esrally/track/loader.py

```diff
@@ -314,7 +314,7 @@ def operation_parameters(t, op):
     if op.param_source:
         return params.param_source_for_name(op.param_source, t, op.params)
     else:
-        return params.param_source_for_operation(op.type, t, op.params)
+        return params.param_source_for_operation(op.type, t, op.params, op.name)
```

Comment:
Each operation will have a name but if we specify operations inline with a task, it's the operation type's name. How about we instead use the task's name here which is always recognizable by the user (see attached [patch](https://github.com/elastic/rally/files/4070975/task_name.patch.txt) for a sketch).